src/lib/staged_ledger/dune

```diff
@@ -2,7 +2,7 @@
  (name staged_ledger)
  (public_name staged_ledger)
  (library_flags -linkall)
- (flags :standard -short-paths -warn-error -58)
+ (flags :standard -short-paths -warn-error -58-32)
```

Comment:
because `version` is unused?

you can add `unnumbered` to the `deriving version` (I think, haven't tried using it with `asserted`)